### ISSUE TITLE
Teaclient old bug

### ISSUE THREAD

**Issue by cyberlimbo** [TeaSpeak/TeaSpeak#683]
Deleting CH massages not working properly
(after deleting the messages, if you disconnect and reconnect the client, some messages are still present)
Some chat messages can not be deleted too.

*These are old problems, i'm just splitting up every entry into a new single issue.

**Comment by Vafin84** [comment]
> Deleting CH massages not working properly
> (after deleting the messages, if you disconnect and reconnect the client, some messages are still present)
> Some chat messages can not be deleted too.

Yes, this problem really exists. I even once opened a forum topic on this issue.
